### ISSUE TITLE
Implement accessibility info panel

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#562]
# Description
A brief description of each section along with a list of keyboard shortcuts should be provided to the user.

## Action Items

- [ ] Implement 311 Data Accessibility Information bar to appear directly under header.
- [ ] Implement slidedown panel to be displayed when 311 Data Accessibility Information bar is clicked or activated with keyboard. This panel will provide a brief description of each section along with a list of keyboard shortcuts. See mockups.

## Resources

<img width="908" alt="Screen Shot 2020-04-23 at 5 50 47 PM" src="https://user-images.githubusercontent.com/40484278/80163432-00e24a80-858b-11ea-9ee1-711fc1c2fcaa.png">

**Comment by adamkendis** [comment]
@danarchen Can you confirm if this has been eliminated or redesigned?

**Comment by adamkendis** [comment]
UI/UX will discuss possible redesign at this week's meeting.

**Comment by adamkendis** [comment]
New design:

https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A2

![Screen Shot 2020-09-03 at 7 35 10 PM](https://user-images.githubusercontent.com/40484278/92193222-9e446980-ee1c-11ea-956b-5e2e4847be4d.png)
